### DISCUSSION TITLE
Question about intergrate Chatgpt/OpenAI

### DISCUSSION THREAD

**Discussion by shixin-guo** [tolgee/tolgee-platform#1861] [General]
I reviewed the code base, but I couldn't find any references to OpenAI or ChatGPT. However, I did come across a blog post where we announced the release of Tolgee Context in version 5.9.0. You can find the blog post here: [Tolgee AI Translator Release](https://tolgee.io/blog/tolgee-ai-translator).

With that in mind, I'd like to confirm whether we have open-sourced this particular section of the code.

**Comment by JanCizmar** [comment]
Hello! Thanks for the question. This part of the code is not open-sourced, and we don't plan to open-source it. However, we plan to enable the Tolgee Translator API in the future for self-hosted instances. 

You can benefit from contextual translation features only on the Tolgee Cloud.

**Comment by shixin-guo** [comment]
Got , Thank you for your prompt response
